### DISCUSSION TITLE
Error on deployment

### DISCUSSION THREAD

**Discussion by danimalx** [Infocaster/UrlTracker#111] [Q&A]
Installed V10 on Umbraco 10 and works fine locally. But when I publish the site live all i see is this...
<img width="427" alt="Screenshot 2023-02-02 at 05 32 13" src="https://user-images.githubusercontent.com/349216/216240123-37e808a1-1e87-4a7f-9c82-91ab1f713f23.png">

**Comment by D-Inventor** [comment]
Hi @danimalx ! Thank you for reaching out!

That's remarkable. Usually you get this when the static assets aren't properly published to the live site or when smidge is unable to serve all the static assets. Do you get any relevant errors in your Umbraco logs?

**Comment by danimalx** [comment]
Sorry for delayed reply! As far as I can see all static assets have been published to wwwroot/app_plugins


I can't see any errors

![image](https://user-images.githubusercontent.com/349216/222896131-62ec1c42-baf0-4833-a093-e55964faae5e.png)

**Reply by D-Inventor** [reply]
I see. There are 2 folders named App_Plugins. The one you found is in wwwroot. There should also be an App_Plugins folder in the application root. Could you check that folder and make sure you delete any URL Tracker files and folders from there? They may have been left there from a previous version of the URL Tracker.

**Comment by danimalx** [comment]
Nothing in there. It's the first install.

D

On Sat, 4 Mar 2023 at 14:54, Dennis ***@***.***> wrote:

> I see. There are 2 folders named App_Plugins. The one you found is in
> wwwroot. There should also be an App_Plugins folder in the application
> root. Could you check that folder and make sure you delete any URL Tracker
> files and folders from there? They may have been left there from a previous
> version of the URL Tracker.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Infocaster/UrlTracker/discussions/111#discussioncomment-5202370>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AACVIIAKFLGZKWHJQZZWGSTW2NJQ5ANCNFSM6AAAAAAUOSIM3E>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
*Dan Evans*

Unit Z, 23 Alphabet Mews, London, SW9 0FN
*t: *07799 693983
*w: *plan9.co.uk

**Reply by D-Inventor** [reply]
Remarkable, I don't know why it would do this then. How about the DevConsole, do you get any errors in there?
